lib/paloma.js

```diff
@@ -27,6 +27,7 @@ module.exports = class Paloma extends Koa {
   }
 
   route(route) {
+    route.engine = route.engine || this.engine || undefined;
```

Comment:
I think this line should be moved to https://github.com/palomajs/paloma-router/blob/master/lib%2Findex.js#L13